lib/commands/web.js

```diff
@@ -8,6 +9,44 @@ let extensions = {};
 
 Object.assign(extensions, iosCommands.web);
 
+extensions.getExtraNativeWebTapOffset = async function () {
+  let offset = 0;
+
+  // first try to get tab offset
+  try {
+    let el = await this.findNativeElementOrElements('-ios predicate string', `name LIKE '*, Tab' AND visible = 1`, false);
+    // this height is either there or not, so cache
+    if (!util.hasValue(this._webTabBarHeight)) {
```

Comment:
I don't think that would work. `_.memoize` caches based on input value, which in this case would have to be the element, which would be different on every call. The element finding can't be inside the memoized function, since then it would only be called once, and it can be either present or absent on any given call.